egs2/TEMPLATE/enh_asr1/scripts/feats/feat_to_shape.sh

```diff
@@ -0,0 +1 @@
+../../../../../utils/feat_to_shape.sh
```

Comment:
It would be better to link the `feats` directory to [egs2/TEMPLATE/asr1/scripts/feats](https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE/asr1/scripts/feats).
Because that would automatically reflect possible changes in the `asr1` template in the future.